docs/overview.yaml

```
@@ -61,7 +61,7 @@ General algebra:
     ideal of a commutative ring: 'ideal'
     quotient ring: 'ideal.has_quotient'
     prime ideal: 'ideal.is_prime'
-    maximal ideal: 'ideal.is_maximal'
+    maximal ideal (as maximal submodule): 'submodule.is_maximal'
```

Maybe
```suggestion
    maximal ideal (generalized to submodules): 'submodule.is_maximal'
```
Let's see how @PatrickMassot would like this written

---

I think it's probably clearer to keep "maximal ideal" there. People clicking this link may be confused for a few seconds but they will very quickly read the docstring.

---

I agree with keeping "maximal ideal" in docs since it's in "Ideals and quotients" sections and shorter is better, so I removed `(as maximal submodule)`.